### ISSUE TITLE
BROKEN METHODS: 3605f REMOVE SESSION ID FROM SESSION INFO FOR PRIVACY!

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#773]
Broken methods detected. Details below. 

<details>
<summary>data</summary>

```javascript
{
  "WA_VERSION": "2.2035.14",
  "PAGE_UA": "WhatsApp/2.2029.4 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36",
  "WA_AUTOMATE_VERSION": "2.0.12",
  "BROWSER_VERSION": "HeadlessChrome/85.0.4182.0",
  "SESSION_ID": "session1",
  "BROKEN_METHODS": [
    "Store.SendMessage.addAndSendMsgToChat"
  ],
  "occurances": 8,
  "lastReported": "Fri Sep 04 2020 23:57:45 GMT+0000 (Coordinated Universal Time)"
}
```

</details>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.13](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.12...2.0.13) (2020-09-05)

- build(deps): bump bl from 4.0.2 to 4.0.3 [`#766`](https://github.com/open-wa/wa-automate-nodejs/pull/766)
- fix #773 [`#773`](https://github.com/open-wa/wa-automate-nodejs/issues/773)
- Release 2.0.13 [`fe33712`](https://github.com/open-wa/wa-automate-nodejs/commit/fe33712f64fe450e76585cbbb127c643369f66ee)
- better typing for initializer params [`76027b7`](https://github.com/open-wa/wa-automate-nodejs/commit/76027b75032612944a72110457fb4bda99fd0e22)
- Fix: `sendLinkWithAutoPreview` now returns id when message sent to groups #760 [`40ce097`](https://github.com/open-wa/wa-automate-nodejs/commit/40ce097b7ef8ddf3ba77e18f15a39cee0cc6e306)
- Fixed optional string type for first param [`16b7dd4`](https://github.com/open-wa/wa-automate-nodejs/commit/16b7dd48c99e0fe3474376b977fd96a568f9e444)
